### ISSUE TITLE
[FEATURE] Basic animations

### ISSUE THREAD

**Issue by alexvgjm** [alexvgjm/point-it-out#35]
**Is your feature request related to a problem? Please describe.**
There are currently no animations and there should be some basic ones.

**Describe the solution you'd like**
 Users should be able to define its own animations eventually by applying CSS to target pointer but it would be great some basic ready-to-go solution like this:
```ts
create('arrow', {
    ...,
    animate: {...} // an AnimationOptions object
})
```

Where AnimationOptions could be something like:

```ts
{
   name: 'pulse', // options based on pointer type
   duration?: number,

   /** equivalent to iteration-count */
   repeat?: number | 'infinite'
}
```

or just animation name assuming default values:
```ts
create('arrow', { 
    ...
    animate: 'pulse'
})
```

**Comment by alexvgjm** [comment]
Thinking out loud...

the most trickiest part is how and where dynamically add the animation `@keyframes`. It can't be inlined and I find the `<style>` in `<head>` injection a bit invasive. It is clear that we need the `<style>` in a scoped level near the raw pointer, but where?

Regarding the animations, I don't recommend specifying and testing through E2E tests. Animations are not as deterministic or easy to specify as the other options are.

**Comment by alexvgjm** [comment]
Finally used a style in head injection.  This way you have a single location where you can find all the keyframes. Injecting elsewhere doesn't make much sense because in any case the scope of the style tag is global.

Implemented in #37
